// model-handler/bundles/model-handler.umd.js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core')) :
    typeof define === 'function' && define.amd ? define('model-handler', ['exports', '@angular/core'], factory) :
    (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global['model-handler'] = {}, global.ng.core));
}(this, (function (exports, i0) { 'use strict';

    var ModelHandlerService = /** @class */ (function () {
        function ModelHandlerService() {
        }
        return ModelHandlerService;
    }());
    ModelHandlerService.ɵprov = i0.ɵɵdefineInjectable({ factory: function ModelHandlerService_Factory() { return new ModelHandlerService(); }, token: ModelHandlerService, providedIn: "root" });
    ModelHandlerService.decorators = [
        { type: i0.Injectable, args: [{
                    providedIn: 'root'
                },] }
    ];
    ModelHandlerService.ctorParameters = function () { return []; };

    var ModelHandlerComponent = /** @class */ (function () {
        function ModelHandlerComponent() {
        }
        ModelHandlerComponent.prototype.ngOnInit = function () {
        };
        return ModelHandlerComponent;
    }());
    ModelHandlerComponent.decorators = [
        { type: i0.Component, args: [{
                    selector: 'lib-model-handler',
                    template: "\n    <p>\n      model-handler works!\n    </p>\n  "
                },] }
    ];
    ModelHandlerComponent.ctorParameters = function () { return []; };

    var ModelHandlerModule = /** @class */ (function () {
        function ModelHandlerModule() {
        }
        return ModelHandlerModule;
    }());
    ModelHandlerModule.decorators = [
        { type: i0.NgModule, args: [{
                    declarations: [ModelHandlerComponent],
                    imports: [],
                    exports: [ModelHandlerComponent]
                },] }
    ];

    /*
     * Public API Surface of model-handler
     */

    /**
     * Generated bundle index. Do not edit.
     */

    exports.ModelHandlerComponent = ModelHandlerComponent;
    exports.ModelHandlerModule = ModelHandlerModule;
    exports.ModelHandlerService = ModelHandlerService;

    Object.defineProperty(exports, '__esModule', { value: true });

})));
