// model-handler/fesm2015/model-handler.js
import { ɵɵdefineInjectable, Injectable, Component, NgModule } from '@angular/core';

class ModelHandlerService {
    constructor() { }
}
ModelHandlerService.ɵprov = ɵɵdefineInjectable({ factory: function ModelHandlerService_Factory() { return new ModelHandlerService(); }, token: ModelHandlerService, providedIn: "root" });
ModelHandlerService.decorators = [
    { type: Injectable, args: [{
                providedIn: 'root'
            },] }
];
ModelHandlerService.ctorParameters = () => [];

class ModelHandlerComponent {
    constructor() { }
    ngOnInit() {
    }
}
ModelHandlerComponent.decorators = [
    { type: Component, args: [{
                selector: 'lib-model-handler',
                template: `
    <p>
      model-handler works!
    </p>
  `
            },] }
];
ModelHandlerComponent.ctorParameters = () => [];

class ModelHandlerModule {
}
ModelHandlerModule.decorators = [
    { type: NgModule, args: [{
                declarations: [ModelHandlerComponent],
                imports: [],
                exports: [ModelHandlerComponent]
            },] }
];

/*
 * Public API Surface of model-handler
 */

/**
 * Generated bundle index. Do not edit.
 */

export { ModelHandlerComponent, ModelHandlerModule, ModelHandlerService };
